lumigator/python/mzai/schemas/lumigator_schemas/workflows.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+from uuid import UUID
+
+from pydantic import BaseModel
+
+from lumigator_schemas.jobs import JobStatus
+
+
+class WorkflowCreate(BaseModel):
```

Comment:
I assume this would only be a first step, right? The logical replacement would be to include a list of jobcreates to be run in sequence.

Reply:
Yep absolutely! This will be subject to further changes as we migrate the frontend/sdk applications to use workflows.